tutorials/introductory/customizing.py

```diff
@@ -104,15 +120,10 @@
 plt.show()
 
 ###############################################################################
-# .. _matplotlib-rcparams:
```

Comment:
I think I am 👍🏻 on this re-structuring to keep 3 headings (to match the 3 numbered things at the top).  I am 50% sure if you can put more than one target on a section heading, but if not the 3 places we reference it will need to be updated.

Reply:
I've updated the 3 places to reference now `customizing-with-dynamic-rc-settings`